### PR TITLE
add skipLogCapture

### PR DESCRIPTION
**Pull request by KazuCocoa**
https://github.com/appium/appium/issues/12203#issuecomment-466308545

### CODE REVIEW

**Comment by KazuCocoa** [comment]
Need https://github.com/appium/appium-xcuitest-driver/pull/878

File: lib/driver.js [hunk 1]
```diff
@@ -351,7 +351,11 @@ class IosDriver extends BaseDriver {
 
     await isolateSimulatorDevice(this.sim, this.opts);
     this.localConfig = await setLocaleAndPreferences(this.sim, this.opts, this.isSafari(), endSimulator);
-    await this.startLogCapture(this.sim);
+    if (this.caps.skipLogCapture) {
+      logger.info("'skipLogCapture' is set. Skipping starting logs, crash and system.");
```
Comment (mykola-mokhnach) [inline lib/driver.js]: `Skipping starting logs, crash and system.` - this sentence looks confusing. Can we rephrase it?

File: lib/driver.js [hunk 2]
```diff
@@ -368,7 +372,11 @@ class IosDriver extends BaseDriver {
     await utils.setBundleIdFromApp(this.opts);
     await this.createInstruments();
     await runRealDeviceReset(this.realDevice, this.opts);
-    await this.startLogCapture();
+    if (this.caps.skipLogCapture) {
```
Comment (mykola-mokhnach) [inline lib/driver.js]: Please move it to a common method to avoid unnecessary duplication

File: lib/driver.js [hunk 3]
```diff
@@ -685,6 +685,19 @@ class IosDriver extends BaseDriver {
     }
   }
 
+  async setUpLogCapture () {
+    if (this.caps.skipLogCapture) {
+      logger.info("'skipLogCapture' is set. Skipping starting logs, crash and system.");
```
Comment (mykola-mokhnach) [inline lib/driver.js]: Skipping the collection of system logs and crash reports.

**Pull request merged**